server/NewBank.java

```diff
@@ -19,33 +20,45 @@ private NewBank() {
 	}
 
 	private void addTestData() {
-		Customer bhagy = new Customer();
-		bhagy.addAccount(new Account("Main", 1000.0));
-		customers.put("Bhagy", bhagy);
+		newbank.server.Customer bhagy = new newbank.server.Customer("Bhagy", "1234");
+		bhagy.addAccount(new newbank.server.Account("Main", 1000.0));
+		customers.put(bhagy.getUsername(), bhagy);
 
-		Customer christina = new Customer();
-		christina.addAccount(new Account("Savings", 1500.0));
-		customers.put("Christina", christina);
+		newbank.server.Customer christina = new newbank.server.Customer("Christina", "1234");
+		christina.addAccount(new newbank.server.Account("Savings", 1500.0));
+		customers.put(christina.getUsername(), christina);
 
-		Customer john = new Customer();
-		john.addAccount(new Account("Checking", 250.0));
-		customers.put("John", john);
+		newbank.server.Customer john = new newbank.server.Customer("Johh","1234");
+		john.addAccount(new newbank.server.Account("Checking", 250.0));
+		customers.put(john.getUsername(), john);
 	}
 
 	public static NewBank getBank() {
 		return bank;
 	}
 
-	public synchronized CustomerID checkLogInDetails(String userName, String password) {
-		if (customers.containsKey(userName)) {
-			return new CustomerID(userName);
+	public synchronized newbank.server.Customer checkLogInDetails(String username, String password, NewBankClientHandler newBankClientHandler) {
+		if (customers.containsKey(username) && customers.get(username).getPassword().equals(password)) {
+			newBankClientHandler.sendOutput("-Username correct");
+			newBankClientHandler.sendOutput("-Password correct");
+			return customers.get(username);
+		}
+		else if (!customers.containsKey(username)){
+			newBankClientHandler.sendOutput("-Wrong username\n");
+			return null;
+		}
+		else if (!customers.get(username).getPassword().equals(password)) {
+			newBankClientHandler.sendOutput("-Wrong password\n");
+			return null;
+		}
+		else {
+			return null;
 		}
-		return null;
 	}
 
-	// commands from the NewBank customer are processed in this method
-	public synchronized String processRequest(CustomerID customer, String request, NewBankClientHandler newBankClientHandler) {
-		if (customers.containsKey(customer.getKey())) {
+	// Commands from the NewBank customer are processed in this method.
+	public synchronized String processRequest(newbank.server.Customer customer, String request, NewBankClientHandler newBankClientHandler) {
```

Comment:
The `newbank.server` is not necessary, and actually makes the code harder to read, please remove it